### ISSUE TITLE
Disable Naming/HeredocDelimiterNaming

### ISSUE THREAD

**Issue by twe4ked** [testdouble/standard#74]
I wrote the following code that contains a heredoc. I don't think changing the name here would add anything. I read the `EOF` like I read `"` in a string.

```ruby
File.open("db/migrations/#{Time.now.utc.strftime("%Y%m%d%H%M%S")}_migration.rb", "w") do |f|
  f.write(<<~EOF)
    Sequel.migration do
      # ... snip
    end
  EOF
end
```

Potential reasons to keep it:

- Adds context to the code
- Some editors can use the name to determine how to highlight the heredoc 

Reasons to disable:

- You can add context to the heredoc in other ways, for instance you could assign the heredoc to a well named variable
- `--fix` can't automatically fix it meaning developers need to spend time updating code

Thanks!

**Comment by JoshCheek** [comment]
I'd suggest changing the example, it undermines your position because it falls under point number 2, _"Some editors can use the name to determine how to highlight the heredoc"._ In this case, the delimiter would be better as `RUBY` 

![image](https://user-images.githubusercontent.com/77495/52446170-d2499280-2af2-11e9-94d3-1885762324e8.png)

---

Here is one from one of my codebases. How would Standard prefer I write this? It's not clear to me, but seeing the answer might make it easier to comprehend this setting. When RubCop hit me with it, I just changed it to arbitrary characters since that seemed better than calling it `SQL` or something else that it isn't.

```ruby
RSpec.describe '...' do
  it '...' do
    # ... some code that makes an http request or smth ...
    expect(response.status).to eq(status), <<~EOS
      Expected response status to be #{status}, but was #{response.status}
      The body was:
      #{JSON.parse(response.body) rescue response.body}
    EOS
  end
end
```

Anyway, like @twe4ked, I also read it like double quotes. We don't require double quoted strings be converted to heredocs and given meaningful names, so if it's okay for those strings, why is it not okay for heredocs?

![screen shot 2019-02-07 at 5 39 36 pm](https://user-images.githubusercontent.com/77495/52450874-86064e80-2b02-11e9-9ba3-c20ff781234f.png)

**Comment by twe4ked** [comment]
> I'd suggest changing the example, it undermines your position because it falls under point number 2

I'll leave it so I don't make the conversation hard to follow. Thanks for your further examples. :)

**Comment by searls** [comment]
Agree, this cop isn't valuable. (It's just here because it's a default I never ran into.)

**Comment by twe4ked** [comment]
Cheers.
